cli/src/commands/mod.rs

```diff
@@ -545,6 +548,82 @@ struct NewArgs {
     insert_before: bool,
 }
 
+/// Move the current working copy commit to the next child revision in the
+/// repository.
+///
+///
+/// The command moves you to the next child in a linear fashion.
+///
+///
+/// D        D  @
+/// |        | /
+/// C  @ =>  C
+/// | /      |
+/// B        B
```

Comment:
nit: The horizontal in the tree diagrams below are slightly inconsistent:

- `next`: 1 space before, 2 spaces after the arrow
- `next --edit`: 2 spaces before, 1 space after the arrow
- `prev`: 1 space before, 2 spaces after the arrow
- `prev --edit`: 0 spaces before, 3 spaces after the arrow

Reply:
Should be fixed now.